### PR TITLE
[13.0][MIG] account_payment_multi_deduction

### PR DESCRIPTION
**Pull request by kittiu**
Standard migration, ready for review.

### CODE REVIEW

**Comment by kittiu** [comment]
@newtratip can you help review / test this?
Thanks!

File: account_payment_multi_deduction/README.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
+========================================
```
Comment (newtratip) [inline account_payment_multi_deduction/README.rst:1]: README still be v12, please update readme to v13

**Comment by newtratip** [comment]
@kittiu 
I put the deduction amount to 0. it create journal items, I think it should not be created ?
Selection_033
Selection_034

File: account_payment_multi_deduction/README.rst (hunk 1 above)
Comment (kittiu) [reply account_payment_multi_deduction/README.rst:1]: Don't need to do README, it will be auto generated (replace) when merged by ocabot.

**Comment by kittiu** [comment]
> @kittiu
> I put the deduction amount to 0. it create journal items, I think it should not be created ?
> Selection_033
> Selection_034

This is intentional, for flexibility it is ok.

File: account_payment_multi_deduction/README.rst (hunk 1 above)
Comment (newtratip) [reply account_payment_multi_deduction/README.rst:1]: OK

**Comment by newtratip** [comment]
Functional Test OK :+1:

**Review by newtratip — APPROVED** [review]
Functional Test OK :+1:

File: account_payment_multi_deduction/readme/HISTORY.rst [hunk 2]
```diff
@@ -1,3 +1,9 @@
+13.0.1.0.0 (2020-01-27)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* Migration to version 13
+* Following how Odoo 13 works wit deduction, the multiple deduction feature only work with 1 invoice at a time.
```
Comment (dreispt) [inline account_payment_multi_deduction/readme/HISTORY.rst]:
```suggestion
* Following how Odoo 13 works with deductiosn, the multiple deduction feature only works for one invoice at a time.
```

**Review by dreispt — COMMENTED** [review]
Minor description text improvements

File: account_payment_multi_deduction/readme/HISTORY.rst (hunk 2 above)
Comment (kittiu) [reply account_payment_multi_deduction/readme/HISTORY.rst]: Thanks @dreispt , fixed.

**Review by dreispt — APPROVED** [review]

**Pull request merged**